<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="helpButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;В данном окне осуществляется работа с изображением эталонной детали. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Перед началом работы необходимо загрузить изображение эталонной детали, задать реперные точки и наложить идеальный контур с помощью соответсвующих пунктов в меню &amp;quot;Настройки&amp;quot;. Всё это необходимо для получения контуров допуска +/-. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;При врещении колёсика мыши при наложении идеального контура происходит повотот изображения на 1/10 градуса. Если во время вращения нажата правая кнопка мыши, происходит вращение на угол в 10 раз больший, если клавиша Shift - на угол в 100 раз больший, если Ctrl - в 10 раз меньший. После того, как идеальный контур совмещён, его нужно зафиксировать двойным щелчком.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="ImageGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Изображение эталонной детали, не загружено</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="main_menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="angle_menu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Ракурс</string>
     </property>
     <addaction name="angle_1_action"/>
     <addaction name="angle_2_action"/>
     <addaction name="angle_3_action"/>
     <addaction name="angle_4_action"/>
     <addaction name="angle_5_action"/>
    </widget>
    <widget class="QMenu" name="load_menu">
     <property name="title">
      <string>Загрузить изображение</string>
     </property>
     <addaction name="load_etalon_1_action"/>
     <addaction name="load_etalon_2_action"/>
     <addaction name="load_etalon_3_action"/>
     <addaction name="load_etalon_4_action"/>
     <addaction name="load_etalon_5_action"/>
     <addaction name="load_current_action"/>
    </widget>
    <widget class="QMenu" name="goto_menu">
     <property name="title">
      <string>Перейти к </string>
     </property>
     <addaction name="goto_current_window_action"/>
     <addaction name="goto_counturs_window_action"/>
    </widget>
    <widget class="QMenu" name="settings_menu">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="settings_action"/>
     <addaction name="set_etalon_reference_points_action"/>
     <addaction name="set_current_reference_points_action"/>
     <addaction name="impose_ideal_contour_to_etalon_action"/>
    </widget>
    <widget class="QMenu" name="shoot_and_load_menu">
     <property name="title">
      <string>Сделать фотографию и загрузить</string>
     </property>
     <addaction name="shoot_and_load_etalon_action"/>
     <addaction name="shoot_and_load_current_action"/>
    </widget>
    <addaction name="load_menu"/>
    <addaction name="shoot_and_load_menu"/>
    <addaction name="angle_menu"/>
    <addaction name="goto_menu"/>
    <addaction name="settings_menu"/>
    <addaction name="save_current_action"/>
    <addaction name="save_result_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="main_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="angle_1_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="angle_2_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="angle_3_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="angle_4_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="load_etalon_1_action">
   <property name="text">
    <string>Эталон ракурса 1</string>
   </property>
  </action>
  <action name="load_etalon_2_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Эталон ракурса 2</string>
   </property>
  </action>
  <action name="load_etalon_3_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Эталон ракурса 3</string>
   </property>
  </action>
  <action name="load_etalon_4_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Эталон ракурса 4</string>
   </property>
  </action>
  <action name="load_current_action">
   <property name="text">
    <string>Текущая деталь</string>
   </property>
  </action>
  <action name="goto_current_window_action">
   <property name="text">
    <string>Окно изображения текущей детали</string>
   </property>
  </action>
  <action name="goto_counturs_window_action">
   <property name="text">
    <string>Окно контуров изображений</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="save_result_action">
   <property name="text">
    <string>Сохранить результат(контуры деталей)</string>
   </property>
  </action>
  <action name="settings_action">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="set_etalon_reference_points_action">
   <property name="text">
    <string>Задать реперные точки для эталона</string>
   </property>
  </action>
  <action name="save_current_action">
   <property name="text">
    <string>Сохранить изображение текущей детали</string>
   </property>
  </action>
  <action name="angle_5_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="load_etalon_5_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Эталон ракурса 5</string>
   </property>
  </action>
  <action name="set_current_reference_points_action">
   <property name="text">
    <string>Задать реперные точки для текущей детали</string>
   </property>
  </action>
  <action name="set_skeleton_inner_points_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Задать точки внутреннего остова для эталона</string>
   </property>
  </action>
  <action name="impose_ideal_contour_to_etalon_action">
   <property name="text">
    <string>Наложить идеальный контур на эталон</string>
   </property>
  </action>
  <action name="shoot_and_load_etalon_action">
   <property name="text">
    <string>Эталона</string>
   </property>
  </action>
  <action name="shoot_and_load_current_action">
   <property name="text">
    <string>Текущей детали</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>imagewindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
